app/models/fields/custom_field.rb

```diff
@@ -49,6 +49,8 @@
 #   since changing the type of the database column would cause data to be lost.
 #
 
+require_relative 'field'
```

Comment:
I assume this is something to do with the zeitgiest autoloader changes?

May be worth doing bundle exec zeitgiest:check and submitting all that as a standalone PR; then doing Rails 7 atop that.

Reply:
yeah im not sure why exactly. still working through this. ActiveRecord::Base.descendents end up having far less classes accounted for at any given instance